mmv1/templates/terraform/examples/redis_cluster_cmek.tf.tmpl

```diff
@@ -0,0 +1,60 @@
+resource "google_redis_cluster" "{{$.PrimaryResourceId}}" {
+  name           = "{{index $.Vars "cluster_name"}}"
+  shard_count    = 3
+  psc_configs {
+    network = google_compute_network.consumer_net.id
+  }
+  kms_key = google_kms_crypto_key.kmskey.id
+  region = "us-central1"
+  deletion_protection_enabled = {{index $.Vars "deletion_protection_enabled"}}
+  depends_on = [
+    google_network_connectivity_service_connection_policy.default,
+    google_kms_crypto_key_iam_binding.kms_key_iam_decrypter
+  ]
+}
+
+resource "google_kms_key_ring" "keyring" {
+  name     = "{{index $.Vars "kms_ring_name"}}"
+  location = "us-central1"
+}
+
+resource "google_kms_crypto_key" "kmskey" {
+  name            = "{{index $.Vars "kms_key_name"}}"
+  key_ring        = google_kms_key_ring.keyring.id
+  rotation_period = "7776000s"
```

Comment:
This is what's causing the failure - `rotation_period` causes the request to automatically generate and send a `nextRotationPeriod` timestamp, which changes from run to run (so it's not deterministic & doesn't work with VCR testing). Changing to bootstrapping the KMS Crypto Key should fix this, since it won't have a rotation period and won't be part of the test create / destroy cycle.

Reply:
Understood. Could another option be removing the rotation period?

Reply:
Yes, but we prefer bootstrapping KMS keys because they can't be deleted once created.

Reply:
Understood.